build-system/artifacts/eslint-notice-license-template.txt

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright <%= YEAR %> The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
I think this file should be moved to a more prominent place so it's visible and easily edited if needed. How about moving it to `amphtml/LICENSE_TEMPLATE.txt`, so it sits next to `amphtml/LICENSE`?